### DISCUSSION TITLE
Welcome to MR_SLAM Discussions!

### DISCUSSION THREAD

**Discussion by MaverickPeter** [MaverickPeter/MR_SLAM#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by U-AMC** [comment]
Hello! And thank you for opening the discussion. I am an undergraduate researcher interested in Multi Robot SLAM system and strategy. I hope to seek contributions and collaborations from this discussion session!
